pkg/test/fixtures.go

```diff
@@ -0,0 +1,84 @@
+package test
+
+import "github.com/aerogear/aerogear-app-metrics/pkg/mobile"
+
+func GetEmptyDataMetric() mobile.Metric {
```

Comment:
Any reason not to return a pointer to `mobile.Metric` here (and in the following functions)?

Reply:
Considering it's only used for tests I'd argue perf/memory usage is not a big issue here.

Reply:
I also wanted to make explicit that these would return a new instance instead of a pointer to something existing, due to the possibility of tests eventually modifying them